vars/diffPipeline.groovy

```diff
@@ -63,3 +64,12 @@ def call(lvVersion) {
       }
    }
 }
+
+def call(HashMap<Integer, List<String>> lvVersions) {
+   // The diff script currently expects a 32-bit LabVIEW version.
+   // Get the first 32-bit build version to use for the diff.
+   // Ultimately, we may need to figure out a way to diff in either
+   // bitness, but for now, we will keep it the same.
+   def firstVersionEntry = lvVersions[32][0]
```

Comment:
I had to stare at this double-indexing for a minute before realizing the first index is a map look-up, not an index.